tensorboard/webapp/runs/views/runs_table/runs_table_test.ts

```diff
@@ -2974,6 +2973,150 @@ describe('runs_table', () => {
           expect(action.type).not.toBe(runSelectorSortChanged.type);
         }
       });
+
+      describe('runs data table integration', () => {
```

Comment:
Why are there so many selector overrides for these test in particular? We are just testing that the data is passed through to the correct action.

Reply:
* The `runs-data-table` is only shown if `enableHparamsInTimeSeries` is `true`.
* `getFilteredRenderableRuns` is needed so that the table will have content.
* `getHparamFilterMap` is not needed anymore, thanks for making me take another look at this.
* `getRunsTableHeaders` is needed to ensure hparams are shown.